libraries/AP_Mission/AP_Mission.h

```diff
@@ -142,6 +142,16 @@ class AP_Mission
         uint8_t trigger;        // triggers one image capture immediately
     };
 
+    // Camera Auto Mount Pivoting Oblique Survey (CAMPOS) command
+    struct PACKED CAMPOS_Cmd {
+        float distance;     //distance
+        //uint16_t shutter;   //unused parameter (shutter integration time)
```

Comment:
we normally don't leave comment out code in .. probably best to remove it

Reply:
all right, I left it there as a placeholder for the mavlink common, but, just as is the case with CAM_TRIGG_DIST, ArduPilot does not implement the shutter integration in that way.